pkgs/servers/calibre-web/default.nix

```diff
@@ -18,7 +18,9 @@ python3.pkgs.buildPythonApplication rec {
 
   prePatch = ''
     substituteInPlace setup.cfg \
+        --replace "singledispatch>=3.4.0.0,<3.5.0.0" "" \
```

Comment:
Why did you remove this version constraint completely?

Reply:
> Backport functools.singledispatch from Python 3.4 to Python 2.6-3.3.

https://pypi.org/project/singledispatch/

Reply:
But actually, https://github.com/janeczku/calibre-web/pull/1977